test/metrics/test_pull_request_metrics.py

```diff
@@ -28,7 +28,7 @@ def test_pull_request_acceptance_rate(metrics):
                                      end_date='2019-12-31 23:59:59',group_by='year').iloc[0]['rate'] > 0
 
 def test_pull_request_closed_no_merge(metrics):
-    assert metrics.pull_requests_closed_no_merge(24).iloc[0]['count'] > 0
-    assert metrics.pull_requests_closed_no_merge(24, 21000, begin_date='2018-1-1 00:00:00',
-                                     end_date='2019-12-31 23:59:59', group_by='year').iloc[0]['count'] > 0
+    assert metrics.pull_requests_closed_no_merge(10).iloc[0]['pr_count'] > 0
+    # assert metrics.pull_requests_closed_no_merge(10, 20, begin_date='2018-1-1 00:00:00',
```

Comment:
@ccarterlandis I fixed the tests & routes. This is the only case that I'm not able to debug as to why I'm receiving an `index out of bounds` error. Would appreciate it if you could take a look!

Reply:
Haha never mind! I was using the wrong repo_id to test! It's working perfectly alright 😃